dashboard/templates/base.html

```diff
@@ -64,24 +56,4 @@
     });
 </script>
 {% endif %}
-<script src="{% static 'moment/min/moment.min.js' %}"></script>
-<script src="{% static 'jquery/dist/jquery.min.js' %}"></script>
-<script src="{% static 'angular/angular.js' %}"></script>
-<script src="{% static 'underscore/underscore-min.js' %}"></script>
-<script>
-    $(document).ready(function () {
```

Comment:
This Help Link code will need to be re-added to the frontend too in #528 but good to remove here. 

The link should really also be provided via a setting.

Reply:
<img width="663" alt="Screenshot 2019-05-30 10 01 37" src="https://user-images.githubusercontent.com/8836578/58649808-f4456b80-82c1-11e9-9c16-a0e3d0166748.png">
The help is actually already linked. But ya the link should be configurable, also in env variable?

Reply:
Yeah right in AvatarModel it looks like it's hardcoded to the umich.edu help. This will need to be an env for like BASE_HELP_URL or something. 

Also the help in base.html appended the specific view this was on to give it "context sensitive" help. I'm not sure how easy that would be in React to have that happen.

Reply:
This should be pretty straight forward